chalice/deploy/packager.py

```diff
@@ -405,6 +415,7 @@ def _download_dependencies(self, directory, requirements_filename):
         # compiler.
         compatible_wheels, incompatible_wheels = self._categorize_wheel_files(
             directory)
+        logger.debug("compatible: %s", compatible_wheels)
```

Comment:
It looks like there are a few `compatible` repeats. It may be good to update it so the statements are unique so it is not ambiguous where the debug line is coming from and worded in a way that I do not have to dive into the code to see at what step the compatible log entry is getting written.

Reply:
Yea so I think this is unavoidable. The logs are only useful if I can see the state of the world between each step.

Reply:
I think Kyle's just asking to add something besides "compatible" so that similar lines (such as line 404) can be distinguished without having to dig through the source code.  Maybe something like `compatible wheels after second download pass`, `compatible wheels after categorization`, etc.